### ISSUE TITLE
When will this software add support for multiple languages?

### ISSUE THREAD

**Issue by ChirsGod** [gitbutlerapp/gitbutler#2831]
Such as Simplified Chinese and other support,And when will the Windows version of the software be released?

**Comment by mtsgrd** [comment]
Hi there! Support for additional languages has not yet been an priority, but it's definitely on our radar. If I were to take a guess right now it's probably a minimum of a couple of months away.

As for Windows support, there is a tracking issue here: https://github.com/gitbutlerapp/gitbutler/issues/2594

**Comment by mtsgrd** [comment]
If you want we can use this as a tracking issue? And perhaps, if you add `i18n` to the title then it will be easier to find using search.

**Comment by ChirsGod** [comment]
Thank you for your reply, I will pay attention to the progress of windows system. The title has also been changed, thank you for your reply to this issue!

**Comment by LaoshuBaby** [comment]
> And perhaps, if you add `i18n` to the title then it will be easier to find using search.

In fact, I was just about to create a request like this (coincidentally, it is also in Simplified Chinese)
I searched **translation** and found that there were no related issues, nor **localization**. Finally, I tried **i18n** as a keyword.

| translation | localization | i18n |
| --- | --- | --- |
| <img src="https://github.com/gitbutlerapp/gitbutler/assets/42690037/69b426ba-7ba1-48db-8dea-9a949c93d0ee" alt="translation"> | <img src="https://github.com/gitbutlerapp/gitbutler/assets/42690037/c2957732-ea77-415b-8317-aabcd6585d6f" alt="localization"> | <img src="https://github.com/gitbutlerapp/gitbutler/assets/42690037/27213eb7-e93a-4c63-8117-22b27ab4ff68" alt="i18n"> |

Besides, if in future you allow community to translate strings, please let us know 

( •̀ ω •́ )✧↑

**Comment by pynickle** [comment]
I hope to see i18n support added as soon as possible. While English is understandable, using one's native language still offers tremendous convenience and comfort.

I really like this project—the UI is very appealing to me. I'm planning to switch from fork to this app. Go ahead!

**Comment by Byron** [comment]
I agree, it would be so great to have. To my mind, earlier is better for these things, at least once everything stabilises.

However,  this is something that's probably going to happen after v1.0 is released, and one can work on more gaining more adoption.

But then again, I am not in charge of priorities, and everything is possible.

**Comment by Byron** [comment]
I have added an official tracking ticket for i18n, closing this in favor of #11723.
